system/lib/wasmfs/file.h

```diff
@@ -69,7 +83,13 @@ class File : public std::enable_shared_from_this<File> {
   std::mutex mutex;
 
 private:
-  // TODO: Add other File properties later.
+  size_t usedBytes = 0;
+
+  uint32_t mode; // r/w/x modes
+
+  time_t ctime; // Time when the inode was last modified
+  time_t mtime; // Time when the content was last modified
+  time_t atime; // Time when the content was last accessed
 
```

Comment:
I wonder if these fields should be on File. That is, perhaps the subtypes should implement them?

For the size, a directory and link won't even need it (directories have none, while links would have the getSize method refer to the target I think).

For the times, I'm less sure... I can imagine file types that get their times from someplace else (like maybe there is an update time on Files in StorageFoundations?). But maybe that's rare.